Classes/Access/Rootline.php

```diff
@@ -86,14 +87,13 @@ class Tx_Solr_Access_Rootline {
 	public function __construct($accessRootline = NULL) {
 		if (!is_null($accessRootline)) {
 			$rawRootlineElements = explode(self::ELEMENT_DELIMITER, $accessRootline);
-
 			foreach ($rawRootlineElements as $rawRootlineElement) {
 				try {
-					$this->push(t3lib_div::makeInstance(
-						'Tx_Solr_Access_RootlineElement',
+					$this->push(\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(
```

Comment:
Please don't mix unrelated changes, it makes reviews unnecessarily harder. Would you mind reverting these kind of changes? We can/will do this in separate commits.